lib/hammer_cli_foreman/exception_handler.rb

```diff
@@ -103,11 +104,20 @@ def ssl_cert_instructions
 
       cmd = "hammer --fetch-ca-cert #{host}"
 
-      _("Make sure you configured the correct URL and have the server's CA certificate installed on your system.") + "\n\n" +
-      _("You can use hammer to fetch the CA certificate from the server. Be aware that hammer cannot verify whether the certificate is correct and you should verify its authenticity after downloading it.") +
-      "\n\n" +
-      _("Download the certificate as follows:") +
-      "\n\n  $ #{cmd}\n\n"
+      if !ssl_option[:ssl_ca_path].nil? || !ssl_option[:ssl_ca_file].nil?
+        instructions = _("The following configuration option were used for the SSL connection:" ) + "\n"
+        instructions << "  ssl_ca_path = #{ssl_option[:ssl_ca_path]}\n" unless ssl_option[:ssl_ca_path].nil?
+        instructions << "  ssl_ca_file = #{ssl_option[:ssl_ca_file]}\n" unless ssl_option[:ssl_ca_file].nil?
+        instructions << "\n" + _("Make sure the location contain valid CA certificate for #{host_url}")
+      else
+        instructions = _("You can use hammer to fetch the CA certificate from the server. Be aware that hammer cannot verify whether the certificate is correct and you should verify its authenticity after downloading it.") +
+            "\n\n" +
+            _("Download the certificate as follows:") +
+            "\n\n  $ #{cmd}\n\n"
```

Comment:
For the sake of consistency, could you use `<<` here too? (btw it's faster than `+`)